src/Provider/Provider.Web/Orchestrators/CloseVacancyOrchestrator.cs

```diff
@@ -44,8 +54,11 @@ public async Task<OrchestratorResponse<VacancyInfo>> CloseVacancyAsync(CloseEdit
 
             Utility.CheckAuthorisedAccess(vacancy, m.Ukprn);
 
-            if (!vacancy.CanClose)
-                throw new InvalidStateException(string.Format(ErrorMessages.VacancyNotAvailableForClosing, vacancy.Title));
+            if (!vacancy.CanClose || true)
```

Comment:
WTF?

Reply:
😱

Reply:
This is a draft PR, please comment on the pattern.

Reply:
Okay. The pattern of checking `if (X || true)` is bad :)